stylesheets/components/image.css

```diff
@@ -0,0 +1,7 @@
+.img-container {
+border:2px solid var(--charcoal-grey);
```

Comment:
I think you don't need to border for responsive images. if you want to border to the particular image then it is okay.

Reply:
I am not giving a border to the images here. As the name suggests, the border is being given to the container that the image is residing in just to show that the image is scaling when viewed across different viewport sizes